client/src/components/Settings/EmailForm.js

```diff
@@ -51,5 +61,12 @@ class EmailForm extends React.Component {
     };
 };
 
+const mapStateToProps = state => {
+    return {
+        fetchingSettings: state.SettingsReducer.fetchingSettings,
+        error: state.SettingsReducer.error,
+        settings: state.SettingsReducer.settings
+    }
+};  
 
-export default withStyles(styles)(EmailForm);
\ No newline at end of file
+export default withStyles(styles)(connect(mapStateToProps, { getSettings })(EmailForm));
```

Comment:
`compose` from `recompose` (https://github.com/acdlite/recompose/blob/master/docs/API.md#compose) might help with this issue :) [I believe I already added when I initially set up the frontend folder]

Something like:
```
const EmailFormPage = compose(
	withStyles(styles),
	connect(mapStateToProps, {getSettings}),
)(EmailForm)

export default EmailFormPage;
```

Reply:
That's awesome! I wasn't sure if we had talked about using recompose as a group, so I'll work it in.